<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>angles</class>
 <widget class="QWidget" name="angles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Mirror&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;(arcsec)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Grating&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;(arcsec)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>top</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Mup</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Gup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Phi</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Psi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>rms</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>bottom</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Mlow</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Glow</string>
   </property>
  </widget>
  <widget class="QCALCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Mrms</string>
   </property>
  </widget>
  <widget class="QCALCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Grms</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>goto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:MonoAktion.PROC</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="PV" stdset="0">
    <string>u411pgm1:MonoAktion.PROC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TouchDiff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>StopDiff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>550</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diff</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>550</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Status</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:touchDiff</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:touchDiffStop</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>570</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:monoGetBmT</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>alpha</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:Alpha</string>
   </property>
   <property name="units" stdset="0">
    <string>deg</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetAlpha</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>alpha</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:Beta</string>
   </property>
   <property name="units" stdset="0">
    <string>deg</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetBeta</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>theta</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:Theta</string>
   </property>
   <property name="units" stdset="0">
    <string>deg</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetTheta</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis4" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>Naverage</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:Naverage</string>
   </property>
   <property name="units" stdset="0">
    <string>deg</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetNaverage</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCALCDNumber</class>
   <extends>QLCDNumber</extends>
   <header>qcalcdnumber.h</header>
  </customwidget>
  <customwidget>
   <class>QCAAxis</class>
   <extends>QWidget</extends>
   <header>qcaaxis.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
